heka/_common.sls

```diff
@@ -34,6 +34,11 @@ heka_acl_log:
 heka_service:
   service.dead:
   - name: heka
+  - enable: false
+  - sig: 'heka.*/etc/heka'
+
+/etc/init.d/heka:
```

Comment:
the service check will still work when you remove service filee as you check the process by sig right?